packages/chain-adapters/src/utils/solanaUtils.ts

```diff
@@ -0,0 +1,9 @@
+import type { SolanaChainId } from '@shapeshiftoss/types'
+
+import { solanaChainIds } from '../solana/SolanaChainAdapter'
+
+export const isSolanaChainId = (
```

Comment:
Sealevel is the underlying protocol of parallelizing smart contracts but SVM (Solana Virtual Machine) looks more near than what we call EVM (https://solana.com/developers/evm-to-svm/smart-contracts)

Reply:
I would call it SVM as they do, and as we do for EVM, changed in [e4891d6](https://github.com/shapeshift/web/pull/8120/commits/e4891d6256da07fae14a9f7eaa0e89ba7aefbe0d)

Reply:
See review in https://github.com/shapeshift/web/pull/8120#pullrequestreview-2456971330 - `isSvmChainId` *is* more correct indeed, however, it may or may not be an early abstraction.
Totally happy with keeping the SVM terminology/abstraction if we wish to, doesn't really hurt to, but we should just be cognicent that chances are that we will never add other SVM ChainIds, meaning that this is one additional layer to reason about here vs. just checking for explicit `SolanaChainId` across the board

Reply:
We kicked this logic from the PR